app/assets/javascripts/application.js

```diff
@@ -4,6 +4,7 @@
 //= require jquery-ui-autocomplete
 //= require modernizr-custom
 //= require dest/respond.min
+//= require raven-3.14.2.min.js
```

Comment:
We could update to the latest sentry 
Also it feels like there would a need to to create a gem to integrate it assets pipeline. We could add this to our backlog, would be a nice and easy open source contribution